volume/fs.go

```diff
@@ -44,7 +45,15 @@ func (d *FS) List(ctx context.Context) ([]plugin.Entry, error) {
 	return List(ctx, d)
 }
 
-var errNonZero = fmt.Errorf("Exec exited non-zero")
+type nonZeroError struct {
```

Comment:
Hmm, why make a structured error object here, but use `plugin.ErrorObject` for the external plugin invoke error?

Reply:
This was a simpler case, I didn't think I'd need more formatting. The error kind is different too. It could make sense to use ErrorObject, but I don't have a case for doing something non-local.

Reply:
Eventually I think I'd like errors to be more complete, so we can do stuff like highlight what part of JSON output produced an error. But we can tackle that separately.

Reply:
I removed the ErrorObj changes. Only difference in output is that the URL help text is at the bottom rather than between error message and detailed output.